tally_ho/apps/tally/models/site_info.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from tally_ho.libs.models.base_model import BaseModel
+
+
+class SiteInfo(BaseModel):
+    site = models.OneToOneField(Site, on_delete=models.PROTECT)
+    user_idle_timeout = models.PositiveIntegerField()
+
+    def __str__(self):
+        return u'%s - %s' % (self.site.name, self.user_idle_timeout)
+
+
+@receiver(post_save, sender=Site)
+def create_site_info(sender, instance, created, **kwargs):
+    if created:
+        SiteInfo.objects.create(site=instance)
+
+
+@receiver(post_save, sender=Site)
+def save_site_info(sender, instance, **kwargs):
+    instance.siteinfo.save()
```

Comment:
I think that it is possible to combine these signals.  But I wonder why you need the second one?  The first one is so that whenever a Site is created, a SiteInfo object is attached to it, but what is the purpose of the second (it just saves SiteInfo whenever Site is saved - why?) ?

Reply:
I think i should rename it to `update_site_info` because it updates `SiteInfo` whenever `Site` is updated.